### DISCUSSION TITLE
AC86U挂载播放卡顿问题

### DISCUSSION THREAD

**Discussion by GTXGOGOGO** [messense/aliyundrive-webdav#488] [Q&A]
华硕AC86U
使用插件版本aliyundrivewebdav-merlin-arm384-armv7.tar.gz（其他版本也都尝试装过了，卡顿还是存在）
固件版本梅林改版386.3-2
网络电信200M
之前是二级路由，挂载正常使用，无线连接也没有卡顿
现在改做拨号路由，kodi播放明显3秒一卡顿，2G视频文件播放也卡顿，关闭打开merlinclash插件也都一样没有改善，电视盒子是神盾2019pro，网线直连
请问可能是啥原因

**Comment by sheldonl3** [comment]
ax86u，外网访问同卡顿，想求解决办法
